apps/framework-cli/src/cli/commands.rs

```diff
@@ -18,12 +18,16 @@ pub enum Commands {
         language: SupportedLanguages,
 
         /// Location of your app or service
-        #[arg(default_value = ".")]
-        location: String,
+        #[arg(short, long)]
```

Comment:
The named parameter `--location` in the docs currently does not work.

```
error: unexpected argument '--location' found

  tip: to pass '--location' as a value, use '-- --location'
```

Reply:
Good catch!